starfish/intensity_table/intensity_table.py

```diff
@@ -428,7 +432,11 @@ def to_expression_matrix(self, regions: Optional[regional.many]=None) -> Express
                 "z": ("cells", np.zeros(counts.shape[0]))
```

Comment:
@ambrosejcarr I was a little confused about this portion, when would we supply the region info?

Reply:
The `regional` objects record cell centroids specifically. If we _don't_ know the centroid, we can guess at it by taking the extreme values of called points -- but if those don't lie at the cell boundaries, the (x, y, z) location of cells won't be properly specified. 

There's another hiccup though: this is out of date. Now we don't use regional, but store segmentation masks.

Reply:
But when do we get the segmentation mask info? Is it a part of SpotAttributes? Or is it something a user already has pre-pipeline?

Reply:
Apologies. we get that from the `Image.Segmentation.Watershed` pipeline component. See #1078 for an updated vignette that includes a usage example for this function.